### PR TITLE
Add Uni the CAT! :DDDD

### PR DESCRIPTION
**Pull request by ghost**
Yes.

### CODE REVIEW

File: launcher/ui/pages/global/LauncherPage.cpp [hunk 1]
```diff
@@ -446,7 +446,10 @@ void LauncherPage::loadSettings()
         ui->themeComboBoxCat->addItem("BackgroundCat");
         ui->themeComboBoxCat->addItem("Jinx");
         ui->themeComboBoxCat->addItem("Floppa");
-        if(currentCatStyle == "Floppa")
```
Comment (slendidev) [inline launcher/ui/pages/global/LauncherPage.cpp:449]: oioioiioioiooiioio don't touch floppa!

Comment (ghost) [reply launcher/ui/pages/global/LauncherPage.cpp:449]: Floppa was just moved down for readability!!!

Comment (slendidev) [reply launcher/ui/pages/global/LauncherPage.cpp:449]: FAKE

**Comment by slendidev** [comment]
Just noticed. Why is this 6 separate commits, two of them ADDING FILES BY UPLOAD!!!!! Squash your commits.

File: launcher/ui/pages/global/LauncherPage.cpp (hunk 1 above)
Comment (ghost) [reply launcher/ui/pages/global/LauncherPage.cpp:449]: Line 452!!!!! Floppa ain't dying son

**Comment by ghost** [comment]
> 

New to this, that's why lol

File: launcher/ui/pages/global/LauncherPage.cpp (hunk 1 above)
Comment (slendidev) [reply launcher/ui/pages/global/LauncherPage.cpp:449]: I can't believe you've done this.

Comment (ghost) [reply launcher/ui/pages/global/LauncherPage.cpp:449]: Do you want floppa to be the last index then? 🤨

Comment (slendidev) [reply launcher/ui/pages/global/LauncherPage.cpp:449]: This is just a joke review lol

Comment (ghost) [reply launcher/ui/pages/global/LauncherPage.cpp:449]: We are all clowns in the same circus

Comment (Mont141414) [reply launcher/ui/pages/global/LauncherPage.cpp:449]: We are all clowns in the same circus

**Comment by glowiak** [comment]
@Mont141414 @xslendix Ehhh, maybe instead of adding cats via commit, would not be better if there was only one cat by default, and an in-launcher option to add custom cat images?

**Comment by ghost** [comment]
> 

In my opinion, that might be a bad thing, as it could be used for other purposes unrelated to cat

**Comment by slendidev** [comment]
I agree, allowing the user to add something else other than a cat (such as a dog), would be against the point of the cat feature.

**Comment by glowiak** [comment]
@xslendix @LennyMcLennington

> I agree, allowing the user to add something else other than a cat (such as a dog), would be against the point of the cat feature.

I don't want rainbows in launcher, but really PolyMC has became a shadow of its former self. Instead of adding new features people are adding new cats (!), Lenny is rarely seen, etc etc.

And the entire community has moved to prism, so that there may be problems with paying for the website (on which the latest year is still 2022).

I am not citicizing, and I understand that you guys do not have time working on this more commonly, but ... well this is the situation.

ehhhhhhhhhhhhhhhhhhhhhhhhhh

**Comment by glowiak** [comment]
and I don't think anyone except me and some people hanging around here use PMC :(

**Comment by funar** [comment]
> and I don't think anyone except me and some people hanging around here use PMC :(

There's more of us than you think :)

**Comment by Mont141414** [comment]
> 

It doesn't have memory leaks like prism though... lol

**Comment by ghost** [comment]
> @xslendix @LennyMcLennington
> 
> > I agree, allowing the user to add something else other than a cat (such as a dog), would be against the point of the cat feature.
> 
> I don't want rainbows in launcher, but really PolyMC has became a shadow of its former self. Instead of adding new features people are adding new cats (!), Lenny is rarely seen, etc etc.
> 
> And the entire community has moved to prism, so that there may be problems with paying for the website (on which the latest year is still 2022).
> 
> I am not citicizing, and I understand that you guys do not have time working on this more commonly, but ... well this is the situation.
> 
> ehhhhhhhhhhhhhhhhhhhhhhhhhh

It doesn't have memory leaks like prism though.... lol

**Comment by HeyaGlitz** [comment]
rip uni pr

**Pull request closed without merging**